openassessment/templates/openassessmentblock/edit/oa_edit_staff_assessment.html

```diff
@@ -0,0 +1,66 @@
+{% load i18n %}
+{% load tz %}
+{% spaceless %}
+<li class="openassessment_assessment_module_settings_editor" id="oa_staff_assessment_editor">
+    <div class="drag-handle action"></div>
+    <div class="openassessment_inclusion_wrapper">
+        <input type="checkbox" id="include_staff_assessment"
+                {% if assessments.staff_assessment %} checked="true" {% endif %}>
+        <label for="include_staff_assessment">{% trans "Step: Staff Assessment" %}</label>
+    </div>
+    <div class="openassessment_assessment_module_editor">
+        <p id="staff_assessment_description_closed" class="openassessment_description_closed {% if assessments.staff_assessment %} is--hidden {% endif %}">
+            {% trans "Staff members assess learners' responses using the rubric for the assignment." %}
+        </p>
+        <div id="staff_assessment_settings_editor" class="assessment_settings_wrapper {% if not assessments.staff_assessment %} is--hidden {% endif %}">
```

Comment:
I'm pretty sure that all that is required for staff grading is a simple checkbox (see wireframes-- https://openedx.atlassian.net/wiki/display/TNL/%5BUX+Wireframes%5D+-+ORA+Staff+Grading). Dates and times don't need to be set.